src/index.scss

```diff
@@ -93,38 +93,42 @@ $white: #fff;
     .nav-link:focus,
     .nav-link.active,
     .expanded .nav-link {
-      background: $component-active-bg;
-      color: $component-active-color;
+      background: var(--pgn-color-bg-active, $component-active-bg);
+      color: var(--pgn-color-active, $component-active-color);
     }
   }
   .main-nav {
     .nav-link {
-      padding: 1.125rem 1rem;
+      padding: 1.125rem var(--pgn-spacing-spacer-base, $spacer);
```

Comment:
This feels like it should be fine, I can't think of any scenario where the `$spacer` value that is defined at the top of this file *wouldn't* be `1rem` by the time this is compiled into `css`.